.github/workflows/test.yml

```diff
@@ -32,11 +32,9 @@ jobs:
         with:
           command: test
           args: --features ${{ matrix.feature }}
+        if: ${{ matrix.feature != '0_18_1' &&  matrix.feature != '0_18_0' && matrix.feature != '0_17_1' }}
       - run: VERSION_WITH_UNDERSCORE=${{ matrix.feature }} && echo "BITCOIND_EXE=../../../.cargo/bitcoin/bitcoin-${VERSION_WITH_UNDERSCORE//_/.}/bin/bitcoind" >> $GITHUB_ENV
       - uses: actions-rs/cargo@v1
```

Comment:
let's remove these lines for now, we may reintroduce the test with BITCOIND_EXE later on